src/extract.rs

```diff
@@ -301,7 +301,7 @@ fn write_rust<W: std::io::Write>(
     section: &Section,
     features: &[Feature],
 ) -> Result<(), std::io::Error> {
-    writeln!(w, "//! {}#{}", target, section.id)?;
+    writeln!(w, "//! {}#section-{}", target, section.id)?;
```

Comment:
You'll need to check if the id is an appendix or not and use the `appendix-` prefix if so.

Reply:
ah thanks for the pointer!

Reply:
on second thought, do appendix sections also contain spec requirements?